setup.cfg

```diff
@@ -112,7 +112,8 @@ install_requires =
     pinax-notifications @ git+https://github.com/GeoNode/pinax-notifications.git@django_upgrade#egg=pinax-notifications
 
     # GeoNode org maintained apps.
-    django-geonode-mapstore-client>=4.0.5,<5.0.0
+    # django-geonode-mapstore-client==4.0.5
+    django-geonode-mapstore-client @ git+https://github.com/GeoNode/geonode-mapstore-client.git@master#egg=django_geonode_mapstore_client
```

Comment:
please @cesar-benjamin do not use git versions inside `setup.cfg` since it breask existing geonode-project (which use their own requirements.txt in turn).
We can consider to adopt your approach, but this must be evaluated in the context of the projects. Maybe in a feature version...

Reply:
i do that because for the geonode-project install method, it install geonode as pip package, so, as python packages dependencies uses setup.cfg and not requirements.txt because the usage of setuptools, so, i have to make chages on setup.cfg to can continue the development and tests, etc. 

On the other hand i can understand the thing about breaks, i guess can be used the releases approach, using a release tag for get a freeze stable release, and because geonode is moving now, geonode-project need to move along with.